packages/react-select/src/components/LiveRegion.tsx

```diff
@@ -77,11 +77,43 @@ const LiveRegion = <
   const ariaSelected = useMemo(() => {
     let message = '';
     if (ariaSelection && messages.onChange) {
-      const { option, removedValue, value } = ariaSelection;
+      const { option, removedValue, removedValues, value } = ariaSelection;
       // select-option when !isMulti does not return option so we assume selected option is value
       const asOption = (val: OnChangeValue<Option, IsMulti>): Option | null =>
         !Array.isArray(val) ? (val as Option) : null;
-      const selected = removedValue || option || asOption(value);
+
+      // we can get an array of options, just a single option or a falsy option,
+      // need to handle returning an Option shaped object or undefined
+      const optionFromPossibleArray = (
+        opt?: Option | Options<Option>
+      ): Option | undefined => {
+        // if falsy return early
+        if (!opt) return;
+        if (Array.isArray(opt)) {
+          // if the array has no items then return early
+          if (!opt.length) return;
+          // take the array of options and reduce to one single option-like object with
+          // the label/s concatenated for the screen reader
+          return opt.reduce(
+            (acc, optionItem: Option) => {
+              const { value: optionVal, label } = optionItem;
+              const sep = acc.label ? ', ' : '';
+              acc.label = acc.label + sep + (label || optionVal);
+              return acc;
+            },
+            // we only need the label for the returned object
+            { label: '' }
+          ) as Option;
```

Comment:
I'm not sure we can create a fake object because it ends up getting passed into `isOptionDisabled` and `getOptionLabel` which is are props that the users can provide.

Reply:
@Methuselah96 Yes, this is a really good call out that I had not considered. I have refactored the implementation to allow using getOptionLabel function. As a consequence I removed this hacky approach which is much nicer.